mypy/checker.py

```diff
@@ -796,6 +796,7 @@ def check_reverse_op_method(self, defn: FuncItem, typ: CallableType,
                 arg_types = list(arg_type.iter_deep())
             else:
                 return
+            # We check that each method is fine when dispatched on a proper self argument
```

Comment:
I understand I should put more info in the commit messages, but this comment is needed regardless. Or are you referring to the code below it?